activation/challenge_verifier.go

```diff
@@ -0,0 +1,181 @@
+package activation
```

Comment:
i am not sure this should be in activation package. i think it should probably stay in grpcserver pkg as this is a generic gateway service

Reply:
It's not generic. It's nipost-challenge-specific. IMO, `grpcserver` package should only contain grpc-related code (parsing input, turning the result into a grpc status etc.) without any business logic. The code in `challenge_verifier.go` is business logic, it needs to know what are the constraints on the nipost challenge etc.